### ISSUE TITLE
Error Column 'tree_left' cannot be null when create a new Taxon in prod environment

### ISSUE THREAD

**Issue by damienBAUDET** [Sylius/Sylius#15164]
**Sylius version affected**: 1.12.8

**Description**  
I get an error when I create a new Taxon in production environment :
`SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'tree_left' cannot be null"`
I don't have any error in development. I tested locally by changing the APP_ENV to prod in the .env.local file and I also get the error. I am using PHP 8.1.12.

**Steps to reproduce**  
Create new project Sylius in a traditional way, don't install demo datas. Change APP_ENV to prod in .env.local. In the administrator interface, try to create a new taxon and you get this error.

Thank you.

**Comment by jakubtobiasz** [comment]
Hello @damienBAUDET 👋🏼

To resolve this issue add the following line to your `composer.json`.
```diff
{
...
"conflict": {
  ...
+"stof/doctrine-extensions-bundle": "1.8.0"
}
...
}
```

So far we thought the problem with the new release of this package exists only on our tests-level, but from what I see it also affects the end-apps. Sylius `1.12.9` will come with this conflict (we already added it), so we'll no longer require to add it manually until we resolve this issue.

P.S. Remember to remove this conflict once we released (and of course you update) Sylius `1.12.9`.
